measures/ResStockArguments/tests/resstock_arguments_test.rb

```diff
@@ -48,7 +48,7 @@ def test_create_geometry_envelope
     arg_name_prefixes = ['geometry', 'door', 'window', 'skylight']
     buildstock_csv_path = File.join(File.dirname(__FILE__), '../../../test/base_results/baseline/annual/buildstock.csv')
     building_ids = (1..(CSV.read(buildstock_csv_path).size - 1)).to_a
-    building_ids = building_ids.sample(0.8 * building_ids.size, random: Random.new(12345)) # 80% for runtime reduction
+    building_ids = building_ids.sample(0.75 * building_ids.size, random: Random.new(12345)) # 75% for runtime reduction
```

Comment:
Trying to keep total CI time ~2 hrs.

Reply:
You should be able to revert this, right? I assume there's no longer an impact on runtime since you are reusing the measure instance.

Reply:
In fact, it should be faster now, if anything, since you are avoiding some duplicate calls to get measure instances.

Reply:
Appears CI time still long for measure-tests. Apparently something newly introduced, probably in `convert_arguments`, over-shadows any savings from measure instances. Any ideas here?